core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -203,6 +210,20 @@ case class Execution[S <: Scheduling](
     hasBeenCancelled
   }
 
+  def forceSuccess()(implicit user: User): Unit = {
+    atomic { implicit txn =>
+      if (forcedSuccess()) {
+        false
```

Comment:
Could we delete it as it isn't used? Just something like this
```
if (!forcedSuccess)
forcedSuccess() = true
print
```

Reply:
Yes sure